tools/plugins/com.liferay.ide.project.ui/plugin.xml

```diff
@@ -316,7 +316,26 @@
            </adapt>
          </enablement>
       </objectContribution>
-
+      <objectContribution
+            adaptable="true"
+            id="com.liferay.ide.project.ui.migrator"
+            objectClass="org.eclipse.core.resources.IProject">
+         <action
+               definitionId="com.liferay.ide.project.ui.migrateProject"
+               class="com.liferay.ide.project.ui.migration.MigrationAction"
+               enablesFor="1"
+               icon="icons/e16/liferay.png"
+               id="com.liferay.ide.project.ui.migrate.project"
+               label="Migrate Liferay plugin project"
+               menubarPath="liferayPopupMenuCategory/slot1"
+               tooltip="Start a migration task for the selected project">
+         </action>
+         <enablement>
```

Comment:
No need for this test, this can work on any normal "project" like a java project even if it isn't a "liferay" project.